Detectors/ITSMFT/MFT/macros/run_trac.C

```diff
@@ -0,0 +1,102 @@
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+
+#include <sstream>
+
+#include <TStopwatch.h>
+
+#include "FairLogger.h"
+#include "FairRunAna.h"
+#include "FairFileSource.h"
+#include "FairRuntimeDb.h"
+#include "FairParRootFileIo.h"
+#include "FairSystemInfo.h"
+#include "Field/MagneticField.h"
+
+#include "MFTReconstruction/TrackerTask.h"
+
+#endif
+
+void run_trac(Int_t nEvents = 10, Int_t nMuons = 100, TString mcEngine = "TGeant3", Float_t rate = 0.)
+{
```

Comment:
Could you please consider interface which takes explicitly input file names as arguments, this legacy interface is not compatible with o2sim output. The ITS macros are also gradually converted to explicit names format (still keeping old interface for bwd compatibility): https://github.com/AliceO2Group/AliceO2/blob/dev/macro/run_trac_its.C